AUTHORS.md

```diff
@@ -11,6 +11,16 @@ Contact:
 
 TODO: Add FelipeRearden, skipadu and vrajur
 
+
+## Vinay Rajur
+ - Contributor
```

Comment:
I forgot to write to the instructions that you can also mention what you have done, e.g. _Created new variable `{{caret_position}}` #114 / #119_. But I can add this mention next time I update the file.

Reply:
No problem, in that case feel free to go ahead and update it :slightly_smiling_face: